### ISSUE TITLE
speech recognition tutorial bug

### ISSUE THREAD

**Issue by ibeling** [tensorflow/tensorflow#21458]
Have I written custom code no
OS Platform and Distribution N/A
TensorFlow installed from N/A
TensorFlow version N/A
Bazel version N/A
CUDA/cuDNN version N/A
GPU model and memory N/A
Exact command to reproduce N/A
Mobile device N/A

I'm doing the Speech Commands tutorial and looking at the low_latency_conv architecture here:
https://github.com/tensorflow/tensorflow/blob/23c218785eac5bfe737eec4f8081fd0ef8e0684d/tensorflow/examples/speech_commands/models.py#L283-L302

The architecture is supposed to be cnn-one-fstride4 from this paper: https://www.isca-speech.org/archive/interspeech_2015/papers/i15_1478.pdf

The paper says "The baseline DNN architecture consists of 3 hidden layers with 128 hidden units/layer and a softmax layer. Each hidden layer uses a rectified linear unit (ReLU) nonlinearity."

However the network generated by the code only contains one ReLU. Subsequently, it just has three stacked MatMul operations with bias, and no interleaved nonlinearities. This is really equivalent to just one MatMul with bias, since you can compose such stacked operations.

Why are the additional ReLUs not included in this network?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by poxvoculi** [comment]
In general the tutorials strive for pedagogical simplicity rather than best-of-breed implementations.  If you'd like to discuss improvements to this model for your purposes, stackoverflow is a better venue.

**Comment by poxvoculi** [comment]
Thanks for tracking down the related issue.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @poxvoculi: It has been 20 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by jvishnuvardhan** [comment]
@ibeling This is a stale issue. 

Since the issue opened, more models were added with two and three `relu` layers along with the initial low-latency model. Depending on the application, different model can be selected.

I think this was resolved. I am closing this issue. Please feel free to reopen if I am mistaken. Thanks!
